### ISSUE TITLE
Rollup badges from Components

### ISSUE THREAD

**Issue by sbtaylor15** [ortelius/ortelius#674]
All component badges plus:

1. Language types, i.e.  45% NodeJS, 15% Python, 40% Golang
2. Risk by Language
3. Repeated Risk - 20% of the components have the same CVE/Risk

**Comment by nikkuv** [comment]
@sbtaylor15  I want to work on this if it is still available.

**Comment by sbtaylor15** [comment]
@nikkuv yes, this should be a good issue to start with.  Let me provide tou some more details on the issue.  I am on holiday this week so pardon the slight delay.

**Comment by nikkuv** [comment]
Ohh... no worries @sbtaylor15 I got busy with something else too but now I can work on this, Can you provide more details now?
